force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +43,13 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> getContactsByFirstName() {
+        List<Contact> contacts = [SELECT Id, FirstName, LastName, Name, Phone, Email 
+            FROM Contact 
+            WHERE FirstName = 'John'];
```

Comment:
Try to use a variable for the string John, also Name, might return the same as FirstName && LastName